src/components/SubMesher.cpp

```diff
@@ -502,18 +502,29 @@ double calculateCrossCoefficient(const Vector& bary_vec, const Vector& tang_vec)
 	return cross[2];
 }
 
+double calculateFaceOrientation(Mesh& mesh, int be)
+{
+	switch (mesh.Dimension())
+	{
+	case 2:
+		return calculateCrossCoefficient(
+			calculateBarycenterVector(mesh, be),
```

Comment:
Similar comment but in this case "vector" is quite generic. Wouldn't be better something like "position"?